### PR TITLE
[Godot tutorial 3D]: Adding new tutorial

### PR DESCRIPTION
**Pull request by GhostNr1**

### CODE REVIEW

File: godot-tutorial-3d/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is an exampel of Godot [Your first 3d game](https://docs.godotengine.org/en/stable/getting_started/first_3d_game/index.html#contents)
```
Comment (Naros) [inline godot-tutorial-3d/README.md]:
```suggestion
This is an example of Godot [Your first 3d game](https://docs.godotengine.org/en/stable/getting_started/first_3d_game/index.html#contents)
```

File: godot-tutorial-3d/project.godot [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+[layer_names]
+
+3d_physics/layer_1="player"
+3d_physics/layer_2="enemies"
+3d_physics/layer_3="world"
+
+[orchestrator]
+
+ui/nodes/highlight_selected_connections=true
```
Comment (Naros) [inline godot-tutorial-3d/project.godot]:
```suggestion
ui/nodes/highlight_selected_connections=false
```
Lets leave this to its default.

**Review by Naros — COMMENTED** [review]
Overall LGTM, just two comments.

**Comment by GhostNr1** [comment]
All fixed

**Pull request merged**